ezgui/src/canvas.rs

```diff
@@ -209,6 +231,21 @@ impl Canvas {
         )
     }
 
+    //the inner bound tells us whether auto-panning should or should not take place
+    pub fn get_inner_bounds(&self) -> Bounds {
```

Comment:
Prefer not `pub` by default; eventually I'd like to clean up all of the ezgui API and release it as a standalone crate. I don't think outside users would need this